apps/frontend/src/hooks/proposal/useDownloadPDFProposal.ts

```diff
@@ -8,8 +8,12 @@ import {
 export function useDownloadPDFProposal() {
   const { prepareDownload } = useContext(DownloadContext);
   const downloadProposalPDF = useCallback(
-    (proposalPks: number[], name: string) => {
-      prepareDownload(PREPARE_DOWNLOAD_TYPE.PDF_PROPOSAL, proposalPks, name);
+    (proposalPks: number[], name: string, downloadType?: string) => {
+      if (downloadType !== undefined) {
+        prepareDownload(PREPARE_DOWNLOAD_TYPE.ZIP_PROPOSAL, proposalPks, name);
+      } else {
+        prepareDownload(PREPARE_DOWNLOAD_TYPE.PDF_PROPOSAL, proposalPks, name);
+      }
```

Comment:
We might need to test the downloadType if it is zip or pdf here since we will be passing it anywhere.